.github/workflows/pull-request.yml

```diff
@@ -272,6 +272,132 @@ jobs:
         with:
           path: './coverage/lcov.info'
           min_coverage: 95.0
+
+  Docker-Check:
+    name: Docker Check
+    needs: Test-Application
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '22.x'
+
+      - name: Cache Node.js dependencies
+        uses: actions/cache@v4
+        with:
+          path: |
+            ~/.npm
+            node_modules
+          key: ${{ runner.os }}-docker-check-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-docker-check-
+
+      - name: Install Docker Compose
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y docker-compose
+
+      - name: Check if Talawa API starts in Docker
+        run: |
+          # Ensure no containers are running
+          docker-compose -f docker-compose.dev.yaml down -v || true
+
+          # Verify docker-compose file exists
+          if [ ! -f "docker-compose.dev.yaml" ]; then
+            echo "Error: docker-compose.dev.yaml not found"
+            exit 1
+          fi
+
```

Comment:
1. Please create a `.github/workflows/scripts` directory
2. Move this to a script located there
3. Reference the script in `.github/workflows/pull-request.yml`

It'll make maintenance easier in future

Reply:
alright will create a new file for the docker script and reference it in this file